### PR TITLE
Initial workflow for Lookup Automox Host from Slack

### PR DESCRIPTION
**Pull request by ax-zyoutz**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [x] Workflow verified as working in a test
- [x] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Triggers, Decisions, and other workflow steps include descriptions where needed
- [x] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [x] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [x] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [x] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

**Comment by ax-zyoutz** [comment]
Workflow and job artifact screenshots:
Screen Shot 2022-02-09 at 8 38 06 PM
Screen Shot 2022-02-09 at 8 37 38 PM
Screen Shot 2022-02-09 at 8 37 32 PM

Slack screenshots for found and not found host:
Screen Shot 2022-02-09 at 8 37 16 PM
Screen Shot 2022-02-09 at 8 36 47 PM

**Comment by ax-zyoutz** [comment]
@blaxminarayan-r7 Absolutely, will get the digits update right now for both slack and teas workflows. As for IPv4/IPv6 support, the endpoint in the action on returns IPv4 addresses at the moment. It's something we hope to follow up with in the future though, just no timeline on the API side yet.

**Comment by ax-zyoutz** [comment]
@blaxminarayan-r7 Commit pushed and tested to refine the IPv4 regular expression with express number of digits

**Review by blaxminarayan-r7 — COMMENTED** [review]
Can you please include the workflow job/url in the slack message, so that you can refer back if needed.

**Review by blaxminarayan-r7 — CHANGES_REQUESTED** [review]
Kindly address the boolean validation and the workflow url

**Review by blaxminarayan-r7 — DISMISSED** [review]

**Review by mberezin-r7 — DISMISSED** [review]

File: workflows/Lookup_Automox_Host_From_Slack/help.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* Fetch host information from Automox quickly in Slack
+* Determine when the system last checked in with the Automox platform
+* Quickly identify the number of upcoming patch count along with patch time 
+* Positively identify a system by looking up its IP or hostname
+
+# Requirements
+
+* Automox API key
+* InsightConnect license
+* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
```
[the remark this answers is not in the document]
Comment (ax-zyoutz) [reply workflows/Lookup_Automox_Host_From_Slack/help.md:17]: Oh, thanks Max. Didn't know there was a new page! Getting this committed now.

**Review by mberezin-r7 — APPROVED** [review]
Thanks Zac!

**Review by blaxminarayan-r7 — APPROVED** [review]

**Pull request merged**